scriptmodules/libretrocores/lr-parallel-n64.sh

```diff
@@ -28,18 +28,20 @@ function sources_lr-parallel-n64() {
     # build from ab155da1 due to https://github.com/libretro/parallel-n64/issues/544
     isPlatform "arm" && commit="ab155da1"
     gitPullOrClone "$md_build" https://github.com/libretro/parallel-n64.git "$branch" "$commit"
+    # avoid conflicting typedefs for GLfloat on rpi4/kms
+    isPlatform "kms" && sed -i "/^typedef GLfloat GLdouble/d" "$md_build/libretro-common/include/glsm/glsm.h"
 }
 
 function build_lr-parallel-n64() {
     rpSwap on 1000
     make clean
     local params=()
-    if isPlatform "rpi" || isPlatform "odroid-c1"; then
-        params+=(platform="$__platform")
-    elif isPlatform "tinker"; then
+    if isPlatform "kms"; then
```

Comment:
Can you change this to `mesa`, please?

IMHO, usage of the `kms` platform flag should be architecture-agnostic when possible due to the fact that any modern system is potentially capable of running in KMS mode. Your PR runs fine on x86 kms mode with the flag changed to `mesa` (whereas the PR breaks the x86 kms build as-is). Thanks.

Reply:
Thanks. Will do.